docs/quickstarts/excel-quickstart-vue.md

```diff
@@ -58,7 +58,7 @@ Each add-in requires a manifest file to define its settings and capabilities.
 
     ![Screenshot of the Yeoman Office Add-in generator command line interface, with project type set to manifest only.](../images/yo-office-manifest-only-vue.png)
 
-After you complete the wizard, it creates a `My Office Add-in` folder, which contains a `manifest.xml` file. You will use the manifest to sideload and test your add-in at the end of the quick start.
+After completion, the wizard creates a `My Office Add-in` folder containing a `manifest.xml` file. You'll use the manifest to sideload and test your add-in.
```

Comment:
Question: is the wizard just the user questions or the installation as well? Wikipedia is sending mixed messages: https://en.wikipedia.org/wiki/Wizard_(software)

Reply:
Has anyone besides me tested this? When I tried, the wizard created a folder with the manifest in it.

Reply:
@lindalu-MSFT  The tool does create the folders. My question is more about the pedantic discussion between the wizard and the tool. It as my understanding that a wizard is only the part of setup that prompts the user, not the part of the installer that actually creates files and folders.